fenix/app/src/main/java/org/mozilla/fenix/library/LibrarySiteItemView.kt

```diff
@@ -41,6 +41,11 @@ class LibrarySiteItemView @JvmOverloads constructor(
 
     val overflowView: ImageButton get() = binding.overflowMenu
 
+    init {
+        overflowView.increaseTapArea(OVERFLOW_EXTRA_DIPS)
```

Comment:
The accessibility issues with this button was resolved on all library screens by https://github.com/mozilla-mobile/firefox-android/pull/1122 which added the padding needed to have the button bigger but visually appear the same.
So this line won't be needed.